packages/perseus/src/widgets/measurer/measurer.tsx

```diff
@@ -161,7 +161,7 @@ class Measurer extends React.Component<Props> implements Widget {
             <div
                 className={
                     "perseus-widget perseus-widget-measurer " +
-                    "graphie-container above-scratchpad"
+                    "graphie-container blank-background"
```

Comment:
Here too, ensuring we don't lose [nested styles](https://github.com/Khan/perseus/pull/2170/files#diff-ff79a578a7a0678d666d4b522615c60cf052a99f70b64109568dd476d9f8e4e4L49)